app/src/services/api.ts

```diff
@@ -48,6 +49,7 @@ export const api = createApi({
     "SectorBreakdown",
     "Inventory",
     "CitiesAndInventories",
+    "Inventories",
```

Comment:
Can we use the Inventory tag we have already here? Normally we don't distinguish between singular/ plural in these tags...

Reply:
I had to add the Inventories so that the list of inventories gets updated without the inventory you just deleted